searx/settings.yml

```diff
@@ -1264,6 +1264,14 @@ engines:
     categories: videos
     disabled : True
 
+  - name: wordnik
+    engine: wordnik
+    shortcut: def
+    base_url: https://www.wordnik.com/
+    categories: general
+    timeout: 5.0
+    disabled: False
```

Comment:
I vote for disabling by default, since wordnik is english only and not all of our users are looking for english terms.

Reply:
Absolutely! Left over from testing. Sorry.